### ISSUE TITLE
setAttributes function

### ISSUE THREAD

**Issue by dyoxyne** [WizUI/WuiDom#56]
It could be nice to have a setAttributes function in the WuiDom prototype.

**Comment by micky2be** [comment]
You can already set attribute on creation (https://github.com/WizUI/WuiDom#using-the-constructor).
If you desire to set/modify the attributes after creation what would you suggest?

**Comment by ronkorving** [comment]
Yeah, that's the point. We have this for everything but attributes. Annoying when dealing with form elements for example (eg: `rootElement.value`).
